doc/ref_impl/cntt-ri2/chapters/chapter01.md

```diff
@@ -56,8 +58,44 @@ The approach this RI will take is to separate out the deployment in to two layer
 
 Meaning, initially, the RI is not looking to have a single installer that can both build out virtual or physical machines **and** build out the Kubernetes and other components. The primary reason for this was to ensure the loose coupling between the two layers, to drive the concept that RI2 is a standalone platform that can (in theory at least) be deployable to any infrastructure, whether that be some physical machines in a lab, or virtual machines in a private or public cloud environment, for example. To combine the deployment of machines and Kubernetes platform with a single installer would potentially limit the locations on which the RI can be installed.
 
-However, that's not to say full end-to-end installers aren't welcome, but if used we must be clear on the limitations and compromises made when we document them in [Chapter 4](./chapter04.md).
+The following subsections provide an overview of the provisioning stages as they are currently planned. However, that's not to say full end-to-end installers aren't welcome, but if used we must be clear on the limitations and compromises made when we document them in [Chapter 4](./chapter04.md).
 
+<a name="1.3.1"></a>
+### 1.3.1 Host Provisioning
+> Add high level description of host provisioning stage
+
+<a name="1.3.2"></a>
+### 1.3.2 Kubernetes Provisioning
+
+For provisioning Kubernetes in the Reference Implementation, existing open source tools should be used to set up the cluster. The goal is to cover the requirements listed in the Kubernetes Based Reference Architecture (RA), but due to the scope this will be done through multiple iterations, starting with a subset of requirements and expanding the deployment over time. 
```

Comment:
```suggestion
For provisioning Kubernetes in the Reference Implementation, existing open source tools should be used to set up the cluster. The goal is to cover the requirements listed in the Kubernetes Based Reference Architecture (RA2), but due to the scope this will be done through multiple iterations, starting with a subset of requirements and expanding the deployment over time. 
```

Reply:
I updated the use of RA/RI to RA2/RI2. In some cases the RI is referenced as "this RI", and I can't decide if those should be updated to "this RI2" as well.